backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -223,6 +224,8 @@ public static void main(String[] args) {
             (Boolean)
                 featureFlagMap.getOrDefault(
                     ModelDBConstants.STORE_CLIENT_CREATION_TIMESTAMP, false);
+        app.setPublicSharingEnabled(
+            (Boolean) featureFlagMap.getOrDefault(ModelDBConstants.PUBLIC_SHARING_ENABLED, false));
```

Comment:
Set parameter are called twice. here and below on `initializeServicesBaseOnDataBase` function So we should have to take one of them.

Reply:
removed from App